src/components/analysis-table.tsx

```diff
@@ -30,7 +30,7 @@ import { EventTeamInfo } from '@/api/event-team-info'
 interface Props {
   eventKey: string
   teams: ProcessedTeamStats[] | undefined
-  schema: Schema
+  schema: Schema | Error
```

Comment:
I think that it would be better to avoid passing in an error as the schema to the analysis table. We can instead wrap the place where we are using the analysis table, with a check to make sure the schema exists before rendering.

If you make that change, then you should be able to undo all of the changes in this file